src/shared/utils/browser/restore-scroll-position.ts

```diff
@@ -1,5 +1,6 @@
 export default function restoreScrollPosition(context: any) {
```

Comment:
Are this and saveScrollPosition still used?

Reply:
Yep, on `Profile`, `Search`, and `Post`.